spec/models/slice/as_json_spec.rb

```diff
@@ -4,8 +4,7 @@
   describe "#as_json" do
 
     let :slice do
-      Slice.new(title: 'Title',
```

Comment:
Ok yeah, this is just in a `wip` commit so not sure what I was thinking. I guess `title` is not a field in `Slice` so it was causing the dynamic attr error, so I just removed it. But actually what we want to do is keep `title` and change `Slice` to `TitleSlice`. :+1: